tensorflow/lite/kernels/internal/reference/softmax.h

```diff
@@ -145,6 +144,21 @@ inline void Softmax(const SoftmaxParams& params,
   }
 }
 
+inline int16_t SoftMaxCalculateExp(const SoftmaxParams& params,
+                                   const int16_t* input_data, const int depth,
+                                   int16_t max_in_row, int i, int c) {
+  int32_t input_diff = input_data[i * depth + c] - max_in_row;
+  // scale the input_diff such that [-65535, 0] correspond to [-10.0, 0.0]
```

Comment:
I know this is existing code so feel free to ignore, but this comment seems stale. Is there something special about the range [-10, 0] that I am missing?

Reply:
I think it's better to keep it.

exp lut generated with range [-10, 0], as exp(-10) is insignificant to accumulate, such logic can be find in `SoftmaxPrepare` function

Reply:
I added a comment regarding @psunn 's explanation.